layers/+lang/go/packages.el

```diff
@@ -131,8 +132,8 @@
 (defun go/init-go-rename()
   (use-package go-rename
     :init
-    (spacemacs/declare-prefix-for-mode 'go-mode "mr" "rename")
```

Comment:
Wouldn't it make sense to simply rename "rename" to "refactor" then add godoctor's shortcuts under it alongside with the current "go-rename"?